packages/kit/src/exports/node/index.js

```diff
@@ -189,3 +191,12 @@ export async function setResponse(res, response) {
 		}
 	}
 }
+
+/**
+ * Converts a file on disk to a readable stream
+ * @param {string} file
+ * @returns {ReadableStream}
+ */
+export function createReadable(file) {
```

Comment:
can we call this something else? I wasn't sure what it was from the name. Maybe `toStream` or `toReadableStream`?

Reply:
`toFoo` sort of implies that it's converting from something to something else (like the `Readable.toWeb` function that it uses under the hood), so I prefer `create`. It can be `createReadableStream` or something if that's preferable though

Reply:
that's fair. `createReadableStream` would be fine with me. it's a little long, but I think that's fine since it's not used super frequently and adds some clarity